arch/arm/core/aarch32/swap_helper.S

```diff
@@ -170,8 +170,8 @@ out_fp_endif:
     /* Restore previous interrupt disable state (irq_lock key)
      * (We clear the arch.basepri field after restoring state)
      */
-#if (defined(CONFIG_CPU_CORTEX_M0PLUS) || defined(CONFIG_CPU_CORTEX_M0)) && \
-	    _thread_offset_to_basepri > 124
+#if (defined(CONFIG_CPU_CORTEX_M0PLUS) || defined(CONFIG_CPU_CORTEX_M0) || \
+	defined(CONFIG_CPU_CORTEX_M1)) && _thread_offset_to_basepri > 124
```

Comment:
In theory, this could be ARMV6_M_ARMV8_M_BASELINE, but I have not tested it in Cortex-M23 :)

Reply:
Yeah, I wondered why it was CPU specific instead of baseline specific. I will leave it as-is until somebody validates it on Cortex-M23.